subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/ReportUncaughtHandler.kt

```diff
@@ -15,7 +15,7 @@ internal class ReportUncaughtHandler(
         if (e.message in nonCriticalErrorMessages) {
             logger.warn("Non critical error caught by ReportUncaughtHandler", e)
         } else {
-            logger.critical("Application crashed", e)
+            logger.warn("Application crashed", e)
```

Comment:
Is it common error?

Reply:
Not really, but it will lead to reported test with `Error` status, which is not much different than any other runtime crash, which is treated more like a developer error, not an infrastructure one